### ISSUE TITLE
loss震荡且map为

### ISSUE THREAD

**Issue by myh12138** [Megvii-BaseDetection/YOLOX#239]
loss经常震荡，而且map为0.

2021-07-29 03:35:16 | INFO     | yolox.core.trainer:262 - epoch: 1/300, iter: 70/142, mem: 28283Mb, iter_time: 2.722s, data_time: 1.849s, total_loss: 16.4, iou_loss: 4.8, l1_loss: 0.0, conf_loss: 10.9, cls_loss: 0.7, lr: 2.430e-05, size: 576, ETA: 1 day, 7:58:24
2021-07-29 03:35:42 | INFO     | yolox.core.trainer:262 - epoch: 1/300, iter: 80/142, mem: 28283Mb, iter_time: 2.634s, data_time: 1.782s, total_loss: 14.9, iou_loss: 4.7, l1_loss: 0.0, conf_loss: 9.3, cls_loss: 0.8, lr: 3.174e-05, size: 544, ETA: 1 day, 7:51:30
2021-07-29 03:36:06 | INFO     | yolox.core.trainer:262 - epoch: 1/300, iter: 90/142, mem: 28283Mb, iter_time: 2.367s, data_time: 1.742s, total_loss: 13.9, iou_loss: 4.7, l1_loss: 0.0, conf_loss: 8.3, cls_loss: 1.0, lr: 4.017e-05, size: 448, ETA: 1 day, 7:25:04
2021-07-29 03:36:39 | INFO     | yolox.core.trainer:262 - epoch: 1/300, iter: 100/142, mem: 28283Mb, iter_time: 3.364s, data_time: 2.160s, total_loss: 22.1, iou_loss: 4.6, l1_loss: 0.0, conf_loss: 16.7, cls_loss: 0.8, lr: 4.959e-05, size: 832, ETA: 1 day, 8:14:28
2021-07-29 03:37:05 | INFO     | yolox.core.trainer:262 - epoch: 1/300, iter: 110/142, mem: 28283Mb, iter_time: 2.520s, data_time: 1.937s, total_loss: 19.7, iou_loss: 5.0, l1_loss: 0.0, conf_loss: 14.4, cls_loss: 0.3, lr: 6.001e-05, size: 672, ETA: 1 day, 8:00:23
2021-07-29 03:37:27 | INFO     | yolox.core.trainer:262 - epoch: 1/300, iter: 120/142, mem: 28283Mb, iter_time: 2.234s, data_time: 1.724s, total_loss: 12.6, iou_loss: 4.5, l1_loss: 0.0, conf_loss: 7.1, cls_loss: 1.0, lr: 7.141e-05, size: 448, ETA: 1 day, 7:31:46
2021-07-29 03:37:52 | INFO     | yolox.core.trainer:262 - epoch: 1/300, iter: 130/142, mem: 28283Mb, iter_time: 2.506s, data_time: 1.792s, total_loss: 15.2, iou_loss: 4.5, l1_loss: 0.0, conf_loss: 9.6, cls_loss: 1.1, lr: 8.381e-05, size: 512, ETA: 1 day, 7:22:18
2021-07-29 03:38:14 | INFO     | yolox.core.trainer:262 - epoch: 1/300, iter: 140/142, mem: 28283Mb, iter_time: 2.225s, data_time: 1.738s, total_loss: 13.9, iou_loss: 4.6, l1_loss: 0.0, conf_loss: 8.3

我用的coco数据集且数据集我验证了一下是对的
[[6.800e+01 7.630e+02 4.740e+02 1.250e+03 1.000e+00]
 [4.900e+02 8.800e+02 8.950e+02 1.089e+03 0.000e+00]
 [4.910e+02 7.100e+01 8.970e+02 2.120e+02 0.000e+00]
 [4.900e+02 1.136e+03 8.950e+02 1.319e+03 1.000e+00]] (1427, 972)
(1427, 972, 3)
68,763,474,1250,2
490,880,895,1089,1
491,71,897,212,1
490,1136,895,1319,2

有大佬觉得这是什么原因吗

**Comment by xxxfrank** [comment]
I have pulled the latest memory leak update and met the same problem too. 
BTW, before that the training process is stable.

**Comment by Gaoyg** [comment]
I met the same problem.

**Comment by VantastyChen** [comment]
> I have pulled the latest memory leak update and met the same problem too.
> BTW, before that the training process is stable.

same problem.

**Comment by ruinmessi** [comment]
For your own custom datasets, I recommend you to first load the COCO pre-trained weights and adopt weak aug (close mixup, change the mosaic scale from [0.1, 2.0] to [0.5, 1.5]. 
If the loss is still unstable, you can try to reduce the initial base lr.

**Comment by myh12138** [comment]
> For your own custom datasets, I recommend you to first load the COCO pre-trained weights and adopt weak aug (close mixup, change the mosaic scale from [0.1, 2.0] to [0.5, 1.5], change the input size range to fit your data.)
> If the loss is still unstable, you can try to reduce the initial base lr.
thanks for your reply, I will try it later.

**Comment by myh12138** [comment]
> For your own custom datasets, I recommend you to first load the COCO pre-trained weights and adopt weak aug (close mixup, change the mosaic scale from [0.1, 2.0] to [0.5, 1.5], change the input size range to fit your data.)
> If the loss is still unstable, you can try to reduce the initial base lr.

Thanks dor your reply, I will try it later. And where to load the COCO pre-trained weights and how to use it to fit this program,?

**Comment by ruinmessi** [comment]
See here https://github.com/Megvii-BaseDetection/YOLOX/blob/main/docs/train_custom_data.md

**Comment by myh12138** [comment]
> See here https://github.com/Megvii-BaseDetection/YOLOX/blob/main/docs/train_custom_data.md

thanks

**Comment by XieLeo11** [comment]
I met the same problem.
